funnel/models/project.py

```diff
@@ -387,17 +392,22 @@ def calendar_weeks(self):
         #         Session.scheduled
         #         ).group_by(db.text('date')).order_by(db.text('date'))
 
+        # if the project is within 2 weeks, send current week as well
```

Comment:
This is incorrect. It checks if the current date is 14 days from the start date. What we want to know is whether the current date is within two weeks (Mon-Fri) of the event's week. If Fifthel starts June 25 (Thursday), then this code will start showing the current date on June 11, which is in the middle of a week. It should start showing from the Monday itself.

Use `isoweek` to get the first day of the week in which the schedule starts, and compare if it's within 14 days of the date on which the current week starts. Or if isoweek supports week math directly, use it. Don't compare week numbers as that will fail around a new year.

Reply:
Looks like you can indeed do `Week.withdate(datetime.now(project.timezone).date()) in (event_week - 1, event_week - 2)`.

I'm not sure, but this may work too: `current_week >= event_week - 2`

Reply:
I can't remember if we had a discussion on `datetime.now(timezone)` vs `utcnow().astimezone(timezone)`. I'm okay with either. The first is standard Python, the second is one less import but one more function call.